mavros_msgs/msg/MountStatus.msg

```diff
@@ -0,0 +1,9 @@
+# Mount Status message
+
+std_msgs/Header header
+
+uint8 target_system
+uint8 target_component
+int32 pointing_a
+int32 pointing_b
+int32 pointing_c
```

Comment:
https://mavlink.io/en/messages/ardupilotmega.html#MOUNT_STATUS

From that definition i do not see any reason to define that message. I believe [Vector3Stamped](http://docs.ros.org/en/noetic/api/geometry_msgs/html/msg/Vector3Stamped.html) is enough, isn't it?